plugins/woocommerce/client/admin/client/settings-payments/components/bank-accounts-list/types.ts

```diff
@@ -1,9 +1,7 @@
 export interface BankAccount {
-	id: string;
```

Comment:
not sure what is the cause, but bank account creation does not work. Maybe due to this id absense?